### DISCUSSION TITLE
Is it possible to fit vertical taskbar with small icon size?

### DISCUSSION THREAD

**Discussion by valdemar-const** [valinet/ExplorerPatcher#2145] [Q&A]
The default windows taskbar width:
![vertical taskbar large icons](https://user-images.githubusercontent.com/34628355/227533532-766d4fb8-9b7f-46f4-86c3-e0854dae2167.png)

The width I want with small icon size:
![vertical taskbar small icons](https://user-images.githubusercontent.com/34628355/227533706-dd58ae67-30ba-4bad-966b-24f3cd439325.png)

**Comment by kong13661** [comment]
It's possible. You can download TaskBar 7+ the tool to manage taskbar for windows 10, and use ExplorePatcher to change the taskbar style to windows 10. Then, in the advance setting of taskbar, change the value of no_limit_width from 0 to 1. New, you can change taskbar to very small width.

**Comment by Aaxfrel** [comment]
Hi, I am on Windows 11 LTSC 2024 and the Windows 10 taskbar is not available. Taskbar Tweaker 7+ by RamenSoftware (which is what I have been using so far to achieve exactly this in other versions of Windows 11) does not seem to be compatible anymore. Is there any other way to achieve the no_limit_width = 1 behavior as with TT7+? Just curious if there are any alternatives.

**Reply by milestprower92** [reply]
some functions from 7+ taskbar tweaker will be available soon on reimplemented taskbar

**Reply by Aaxfrel** [reply]
Sweet! great to hear. Can't wait.

**Reply by moookino** [reply]
do you have any update on this? I use vertical taskbar option and i'd like to make taskbar smaller
